### ISSUE TITLE
Security vulnerability in url-parse, please update module to version >=1.5.8

### ISSUE THREAD

**Issue by nicholeuf** [splitio/javascript-client#656]
https://github.com/advisories/GHSA-hgjh-723h-mx2j

I see the latest version of this library is pulling in version `1.5.4` when I use `yarn why` locally.

```
=> Found "original#url-parse@1.5.4"
info This module exists because "@splitsoftware#splitio#eventsource#original" depends on it.
info Disk size without dependencies: "76KB"
info Disk size with unique dependencies: "120KB"
info Disk size with transitive dependencies: "120KB"
```

**Comment by chillaq** [comment]
Thanks @nicholeuf , will check with the team

**Comment by chillaq** [comment]
Hi @nicholeuf , we released a fix in  SDK version 10.18.0
